combat-trainer.lic

```diff
@@ -981,7 +981,11 @@ class SpellProcess
     @cambrinth_invoke_exact_amount = settings.cambrinth_invoke_exact_amount
     echo("  @cambrinth_invoke_exact_amount: #{@cambrinth_invoke_exact_amount}") if $debug_mode_ct
 
-    @harness_for_attunement = settings.use_harness_when_arcana_locked
+    if settings.combat_trainer_always_harness
```

Comment:
💡 Could this be simplified to `@harness_for_attunement = settings.combat_trainer_always_harness || settings.use_harness_when_arcana_locked`?

Reply:
yes it can, I'm an idiot sometimes :D